File: Proyecto Vecinos Unidos/modelo/evento.py
# modelo/evento.py

from controlador.app import db

class Evento(db.Model):
    __tablename__ = 'eventos'

    id_evento = db.Column(db.Integer, primary_key=True)
    evento = db.Column(db.String(200), nullable=False)
    fec_evento = db.Column(db.Date, nullable=False)
    patrocinadores = db.Column(db.String(200), nullable=True)
    direccion_evento = db.Column(db.String(200), nullable=False)
    precio_evento = db.Column(db.Float, nullable=False)
    capacidad_evento = db.Column(db.Integer, nullable=False)

    def __repr__(self):
        return f"<Evento {self.evento}>"

File: Capstone/Capstone/Models/evento.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Capstone.Modelo
{
    [Table("eventos")]
    public class Evento
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int IdEvento { get; set; }

        [Required]
        [StringLength(200)]
        public  string? NombreEvento { get; set; }

        [Required]
        public DateTime FechaEvento { get; set; }

        [StringLength(200)]
        public string? Patrocinadores { get; set; }

        [Required]
        [StringLength(200)]
        public string? DireccionEvento { get; set; }

        [Required]
        public float PrecioEvento { get; set; }

        [Required]
        public int CapacidadEvento { get; set; }

        public override string ToString()
        {
            return $"{NombreEvento}";
        }
    }
}
